components/sendgrid/actions/search-contacts/search-contacts.js

```diff
@@ -0,0 +1,26 @@
+const sendgrid = require("../../sendgrid.app");
+
+module.exports = {
+  key: "sendgrid-search-contacts",
+  name: "Search Contacts",
+  description: "Searches contacts with a SGQL query.",
+  version: "0.0.4",
```

Comment:
```suggestion
  version: "0.0.1",
```

Reply:
Well, since I am using `pd publish` for testing out my actions, I think Pipedream`s publishing mechanism would fail if we left version `0.0.1`, or lower than the great version used. In a previous review @dylburger suggested, after the PR was approved, to publish components using `0.1.0`. So while I do is as I commit to the repo I will use `0.0.1` but set to `0.10` for merging,. after the PR is approved.

Reply:
You should set the version in the merged PR to `0.0.1` in this case. The version tied to your published component in your account is distinct from the version that will be used to publish, so even if you're on `0.0.4` in your account, you can publish `0.0.1` to the global registry.

Reply:
Gotcha! Will do then.